blocks/subscriptions-block/components/ActionDialog/styles.scss

```diff
@@ -0,0 +1,79 @@
+.xpmedia-action-dialog {
+  display: grid;
+  grid-template-columns: map-get($spacers, 'md') repeat(6, 1fr) map-get($spacers, 'md');
+  grid-template-rows: auto;
+  margin: map-get($spacers, 'xxl') auto;
+  text-align: center;
+  @media screen and (min-width: map-get($grid-breakpoints, 'md')) {
+    margin: map-get($spacers, 'xl') auto;
+  }
+
+  .xpmedia-action-dialog-reason-prompt {
+    color: $ui-medium-gray;
+    grid-column: 3/7;
+    line-height: 1rem;
+    font-size: 0.875rem;
+    @media screen and (min-width: map-get($grid-breakpoints, 'md')) {
+      font-size: 1rem;
+      line-height: 1.5rem;
+    }
+  }
+
+  .xpmedia-action-dialog-link-prompt {
```

Comment:
This and the `.xpmedia-action-dialog-reason-prompt` CSS are the same

Reply:
Merging